Resources/config/services.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="app.twitter_oauth" class="Integrated\Bundle\SocialBundle\Social\Twitter\Oauth">
+            <argument key="twitter_consumer_key">%twitter_consumer_key%</argument>
```

Comment:
These should be configured by a bundle configuration: http://symfony.com/doc/current/bundles/configuration.html